training/src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { Error, Input } from './style';
+
+const TextField = (props) => {
+  const { value, disabled, error } = props;
+  if (error) {
```

Comment:
Optimise the code
If-else block is not necessary

Reply:
done